x/oracle/keeper/keeper_test.go

```diff
@@ -0,0 +1,258 @@
+package keeper
+
```

Comment:
In the future if you could do [TableDrivenTests](https://github.com/golang/go/wiki/TableDrivenTests) that would be very appreciated! :) Improves test readability a lot

Reply:
Looks great, will keep in mind thanks!